mne/channels/tests/test_montage.py

```diff
@@ -920,4 +921,29 @@ def test_make_dig_montage_errors():
     with pytest.raises(TypeError, match='must be an instance of Digitization'):
         _ = DigMontage(ch_names=['foo', 'bar'], dig=None)
 
+# XXX: This comes from test_meas_info
+from mne._digitization._utils import _read_dig_points
+def test_read_dig_polhemus():
+    """Test reading Polhemus data."""
+    extra_points = _read_dig_points(hsp)  # _fname)
+    info = create_info(ch_names=['Test Ch'], sfreq=1000., ch_types=None)
+    assert info['dig'] is None
+
+    # XXX: this shows that we can populate info['dig'] directly.
+    #      Should we forbade that and force `set_montage` ??
```

Comment:
yes ! info['dig'] should be in head coord and enforced during set_montage

Reply:
JFR, info is a dict (`class Info(dict):`) Therefore I'm not sure how much can we tune the insert/setter or whatever is called.

Reply:
what is for sure is that this will not happen in this PR.

Reply:
you cannot prevent users from doing something bad. Just make sure no example or documentation suggests this.

Reply:
JFR; I think we are fine:
```sh
~/code/mne-python read_dig_montage_polhemus*
(mne) ❯ git grep -i "info\['dig'] =" | grep =               
mne/channels/montage.py:            info['dig'] = _make_dig_points(
mne/channels/montage.py:            info['dig'] = montage.dig
mne/channels/montage.py:            info['dig'] = None
mne/channels/tests/test_layout.py:    info['dig'] = info['dig'][:-2]
mne/channels/tests/test_layout.py:    info['dig'] = [d for d in info['dig'] if d['kind'] != FIFF.FIFFV_POINT_EEG]
mne/channels/tests/test_layout.py:    info['dig'] = None
mne/channels/tests/test_layout.py:    info['dig'] = []
mne/gui/_file_traits.py:                info['dig'] = dig
mne/gui/_kit2fiff_gui.py:            raw.info['dig'] = _make_dig_points(self.fid[0], self.fid[1],
mne/io/artemis123/artemis123.py:        info['dig'] = _read_pos(pos_fname)
mne/io/artemis123/artemis123.py:        info['dig'] = []
mne/io/artemis123/artemis123.py:                    self.info['dig'] = _make_dig_points(nasion=nas, lpa=lpa,
mne/io/bti/bti.py:        info['dig'] = None
mne/io/ctf/ctf.py:        info['dig'] = _format_dig_points(info['dig'])
mne/io/ctf/tests/test_ctf.py:            raw.info['dig'] = raw.info['dig'][:-10]
mne/io/kit/kit.py:        info['dig'] = dig_points
mne/io/kit/tests/test_kit.py:    raw_bin.info['dig'] = raw_bin.info['dig'][:8]
mne/io/kit/tests/test_kit.py:    raw_py.info['dig'] = raw_py.info['dig'][:8]
mne/io/meas_info.py:    info['dig'] = _format_dig_points(dig)
mne/io/tests/test_meas_info.py:    info['dig'] = _make_dig_points(extra_points=extra_points)
mne/io/tests/test_meas_info.py:    info['dig'] = _make_dig_points(nasion, lpa, rpa, elp_points[3:], None)
mne/simulation/tests/test_raw.py:    raw_bad.info['dig'] = None
mne/viz/tests/test_3d.py:    info['dig'] = None
mne/viz/tests/test_topomap.py:    evoked.info['dig'] = None
```